### PR TITLE
Remove sentry references

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Remove all sentry references, as it is not used anymore. It was replaced from Posthog, so we can remove all the references

## Feature branch env
[Feature Branch Link](http://fb-chore-remove-sentry-references.fb.qa.budibase.net)

### CODE REVIEW

**Review by mike12345567 — APPROVED** [review]
LGTM - will just wait for confirmation from @aptkingston and @shogunpurple they are happy to get this merged as well!

Just to add to the reasoning for this - we found some errors which are being thrown on startup due to Sentry not being available, since it is no longer used we can simply get rid of the problem rather than fixing it.

**Review by shogunpurple — APPROVED** [review]
LGTM - yeah we get everything we need now generally from datadog. Sentry is a bit too noisy

**Comment by adrinr** [comment]
> LGTM - yeah we get everything we need now generally from datadog. Sentry is a bit too noisy

Just confirming that is not used for the frontend either @aptkingston

File: packages/builder/src/analytics/index.js [hunk 1]
```diff
@@ -23,12 +21,9 @@ class AnalyticsHub {
 
   identify(id) {
     posthog.identify(id)
-    sentry.identify(id)
   }
 
-  captureException(err) {
-    sentry.captureException(err)
-  }
+  captureException(err) {}
```
Comment (adrinr) [inline packages/builder/src/analytics/index.js]: @aptkingston should this not be using posthog?

File: packages/server/src/db/linkedRows/LinkController.ts [hunk 2]
```diff
@@ -434,8 +433,7 @@ class LinkController {
           await this._db.put(linkedTable)
         }
       } catch (err) {
-        /* istanbul ignore next */
-        Sentry.captureException(err)
+        // Ignored error
```
Comment (adrinr) [inline packages/server/src/db/linkedRows/LinkController.ts:436]: @mike12345567 @shogunpurple  should this be using posthog, or is done automatically?

File: packages/builder/src/analytics/index.js [hunk 3]
```diff
@@ -23,12 +21,9 @@ class AnalyticsHub {
 
   identify(id) {
     posthog.identify(id)
-    sentry.identify(id)
   }
 
-  captureException(err) {
-    sentry.captureException(err)
-  }
+  captureException(_err) {}
```
Comment (adrinr) [inline packages/builder/src/analytics/index.js:26]: @aptkingston should this not be using posthog?

File: packages/server/src/db/linkedRows/LinkController.ts (hunk 2 above)
Comment (mike12345567) [reply packages/server/src/db/linkedRows/LinkController.ts:436]: To catch errors with Posthog, we just need to console.error it, or use the `import { logging } from "@budibase/backend-core"` and `logging.logWarn` to log a message + the err object easily in a way that will be detected by Posthog.

File: packages/builder/src/analytics/index.js (hunk 3 above)
Comment (mike12345567) [reply packages/builder/src/analytics/index.js:26]: Posthog isn't really an option in the frontend right now, we don't have the ability to do this. Really we need a new way for detecting this stuff, but any errors that go to Sentry are ignored today, so the above isn't actually a change.

**Pull request merged**